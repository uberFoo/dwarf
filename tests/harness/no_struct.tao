/// This test throws an error.
impl Complex {
    fn add(self, other: Complex) -> Complex {
        Complex {
            re: self.re + other.re,
            im: self.im + other.im,
        }
    }
}
