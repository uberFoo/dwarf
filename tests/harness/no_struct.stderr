[31mError:[0m Unknown type: `Complex`.
   [38;5;246mâ•­[0m[38;5;246mâ”€[0m[38;5;246m[[0m no_struct:2:1 [38;5;246m][0m
   [38;5;246mâ”‚[0m
 [38;5;246m2 â”‚[0m [31mâ•­[0m[31mâ”€[0m[31mâ–¶[0m[31m [0m[31mi[0m[31mm[0m[31mp[0m[31ml[0m[31m [0m[31mC[0m[31mo[0m[31mm[0m[31mp[0m[31ml[0m[31me[0m[31mx[0m[31m [0m[31m{[0m
 [38;5;240m  â”†[0m [31mâ”†[0m   
 [38;5;246m9 â”‚[0m [31mâ”œ[0m[31mâ”€[0m[31mâ–¶[0m[31m [0m[31m}[0m
 [38;5;240m  â”‚[0m [31mâ”‚[0m       
 [38;5;240m  â”‚[0m [31mâ•°[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m[31mâ”€[0m unknown type [36mComplex[0m
[38;5;246mâ”€â”€â”€â•¯[0m

